plugins/builds/README.md

```diff
@@ -8,16 +8,21 @@ const server = new Hapi.Server();
 const buildsPlugin = require('./');
 const Datastore = require('your-datastore-here');
 const Executor = require('your-executor-here');
+const Models = require('screwdriver-models');
+const datastore = new Datastore();
+const executor = new Executor({
+    executorOption1: 'hostname'
+});
 
 server.connection({ port: 3000 });
 
 server.register({
     register: buildsPlugin,
     options: {
-        datastore: new Datastore(),
-        executor: new Executor({
-            executorOption1: 'hostname'
-        })
+        pipelineFactory: Models.PipelineFactory.getInstance({ datastore }),
```

Comment:
Hmm. Maybe? Did `options.datastore` actually work before? Alternatively, we could just remove the options part entirely.